packages/shared/lib/contexts/onboarding/actions/findShimmerRewards.ts

```diff
@@ -161,7 +161,7 @@ export function setTotalUnclaimedShimmerRewards(_totalUnclaimedShimmerRewards: n
 
 function showRewardsFoundNotification(updatedTotalUnclaimedShimmerRewards: number): void {
     const foundRewardsAmount = updatedTotalUnclaimedShimmerRewards - totalUnclaimedShimmerRewards
-    const foundRewardsAmountFormatted = formatTokenAmountBestMatch(foundRewardsAmount, getOnboardingBaseToken())
+    const foundRewardsAmountFormatted = formatTokenAmountBestMatch(foundRewardsAmount, getBaseToken())
```

Comment:
This will break because there is no Node Info loaded in the onboarding, it's only loaded once you log in.